cartographer/mapping_3d/sparse_pose_graph/constraint_builder.cc

```diff
@@ -179,34 +179,36 @@ void ConstraintBuilder::ComputeConstraint(
   // The 'constraint_transform' (submap i <- scan j) is computed from:
   // - a 'high_resolution_point_cloud' in scan j and
   // - the initial guess 'initial_pose' (submap i <- scan j).
+  std::unique_ptr<scan_matching::FastCorrelativeScanMatcher::Result>
+      match_result;
   float score = 0.f;
```

Comment:
Remove these variables? The PR is incomplete: We add the no longer correct values from these unused variables to the histograms.